addons/web/static/src/js/views/control_panel/control_panel_view.js

```diff
@@ -219,6 +219,22 @@ var ControlPanelView = Factory.extend({
 
 
         _.each(preFilters, function (preFilter) {
+            if (preFilter.attrs && JSON.parse(preFilter.attrs.modifiers || '{}').invisible) {
```

Comment:
Might as well tag the `filter` object with the `invisible` attribute here (and remove the check from `_extractAttribute`) to not evaluate the same attribute twice

Reply:
I thought they can can be issues with group count (https://github.com/odoo/odoo/pull/53867/files#diff-7ff998e06f9296609425336c5e3aa496L232). I'll re-test it.

Reply:
I've moved it down and removed related part in `_extractAttributes`.